apps/docs/src/lib/components/TopBar/TopBar.svelte

```diff
@@ -102,8 +131,8 @@
 				on:click={changeModalState}
 			>
 				<div style="display: flex; align-items: center">
-					<MagnifyingGlass size={25} />
-					<p style="margin-left: 0.5rem; font-size: 1.1rem">Search</p>
+					<MagnifyingGlass size={25} color="#228be6" />
+					<h1 style="margin-left: 0.5rem; font-size: 1.1rem">Search</h1>
```

Comment:
Hum, not sure about this, since it's not really a title and it looks similar to the title of the nav. Perhaps `p` here is better

Reply:
Done